### DISCUSSION TITLE
Share your projects!

### DISCUSSION THREAD

**Discussion by FreddieHong19** [FreddieHong19/Open5x#18] [Show and tell]
Share your 5-axis 3D printer and the printed objects here!

**Comment by RedPetroleum** [comment]
There is a further development of the Open5x Prusa currently exhibited at Formnext in Frankfurt, Germany. (Hall 11.0, D32)

**Reply by Itrel** [reply]
Can you send links or pictures?

**Reply by MRackliffe** [reply]
I have a coworker attending, and she was able to send me this. Looks like there are some new shapes, but not necessarily more complex ones. It's hard to tell. ([5-Axis-Printing.pdf](https://github.com/user-attachments/files/17836221/5-Axis-Printing.pdf))
![image](https://github.com/user-attachments/assets/2b84e61b-9d49-41e6-9535-bb0918380a6f)

**Reply by enitmujk** [reply]
Respected @MRackliffe , is there a way of contacting you or the company that worked on the development of the Open5x that is on the picture? My colleague and I are currently working on one, and would like to ask some questions regarding the whole process.  Thank you so much!

**Reply by RedPetroleum** [reply]
> is there a way of contacting you or the company that worked on the development of the Open5x that is on the picture?

It was built by GoetheLab at FH Aachen University of Applied Sciences. Their contact email is: goethelab@fh-aachen.de

**Reply by enitmujk** [reply]
Thank you!
